WC2-527 Add Django admin filters to detect data issues

bramj opened the pull request:
Main additions:
- Show more data on the entities
- New filters on the entities:
	- attributes_id None or not?
	- are the attributes "valid"? This means, if the entity is not deleted, the attributes should not be deleted. Is there a file on the attributes?
	- are there any duplicated uuids?
- New filter on the instances for duplicated uuids
- General performance improvements on several pages, notably on the instances page.

Related JIRA tickets : https://bluesquare.atlassian.net/browse/WC2-527

## How to test

Have some data in your db, look at the Django admin. You can change some things like the uuid in your DB if you want to test the filters.

## Print screen / video

![image](https://github.com/user-attachments/assets/ab7f102d-52c2-4001-b3d0-3f100d18c2d1)

bramj committed:
WC2-527 Add Django admin filters to detect data issues

Main additions:
- Show more data on the entities
- New filters on the entities:
	- attributes_id None or not?
	- are the attributes "valid"? This means, if the entity is not
deleted, the attributes should not be deleted. Is there a file on the
attributes?
	- are there any duplicated uuids?
- New filter on the instances for duplicated uuids
- General performance improvements on several pages, notably on the
instances page.

bramj committed:
Fix: Black

mathvdh reviewed (commented):
All good to me,

Except that I have a visibility problem on the merged_to in admin :

<img width="1157" alt="Screenshot 2024-09-18 at 11 25 24" src="https://github.com/user-attachments/assets/2ae04e37-5253-4d07-85fc-66cfecaa1c40">

I it only because of my local data ??

bramj committed:
WC2-527 Improvement: set raw id input for merged_to...

instead of the selection list.
There can be a large number of results. Also the name that's used as
label is often empty.
Do the same for the ref to entity from the instance admin.

bramj commented:
@mathvdh good catch! It's actually because the `name` on the entity is usually blank. In any case it's best to not use a dropdown here for performance reasons. I've just changed it in the last commit:
![image](https://github.com/user-attachments/assets/9a75cc14-1dbf-42ee-9a9f-96645cb93450)

mathvdh reviewed (approved):
👍🏼
